pkg/ocm/client.go

```diff
@@ -71,6 +71,10 @@ func (l *SdKLogger) Error(ctx context.Context, format string, args ...interface{
 	l.FieldLogger.Errorf(format, args...)
 }
 
+func (l *SdKLogger) Fatal(ctx context.Context, format string, args ...interface{}) {
```

Comment:
Where it is used? fatal log with crash the service.

Reply:
It is not but it was added in the logger interface used in ocm-sdk so in order to use our current logger we need to implement it so we can use it as the required interface.
@filanov